apps/safe-claiming-app/src/components/steps/Claim/SelectedDelegate.tsx

```diff
@@ -16,20 +17,37 @@ export const SelectedDelegate = ({
 }) => {
   const [isHover, setIsHover] = useState(false)
 
+  const isDarkMode = useDarkMode()
   const hasAction = onClick !== undefined
   return (
     <Card variant={hasAction ? "outlined" : "elevation"} elevation={0}>
       <CardHeader
         avatar={<DelegateAvatar selected={false} delegate={delegate} />}
         title={delegate.name}
         subheader={delegate.ens ?? shortenAddress(delegate.address)}
+        sx={{
+          "& .MuiCardHeader-action": {
+            margin: "0px",
+          },
+        }}
         action={
           hasAction && (
             <IconButton
-              color={isHover ? "primary" : undefined}
-              sx={({ palette }) => ({
-                backgroundColor: palette.background.default,
-              })}
+              sx={{
+                color: "text.primary",
+                backgroundColor: ({ palette }) =>
+                  isHover
+                    ? `${palette.safeGreen.light} !important`
+                    : isDarkMode
+                    ? `${palette.background.paper} !important`
+                    : `${palette.border.background} !important`,
+                border: isDarkMode
+                  ? ({ palette }) =>
+                      isHover
+                        ? `1px solid ${palette.safeGreen.main}`
+                        : `1px solid ${palette.text.primary}`
+                  : undefined,
+              }}
```

Comment:
This is a lot of logic for a small design difference. I would suggest either removing it or moving it into the theme overrides.

Reply:
I'll try to somehow combine it in a way that it looks decent in dark and light mode but works the same otherwise